packages/filecoin-client/src/dealer.js

```diff
@@ -93,20 +93,17 @@ export async function aggregateAccept(
 ) {
   /* c8 ignore next */
   const conn = options.connection ?? connection
-
-  const block = await CBOR.write(pieces)
   const invocation = Dealer.aggregateAccept.invoke({
     issuer,
     /* c8 ignore next */
     audience: audience ?? services.AGGREGATOR.principal,
     with: resource,
     nb: {
       aggregate,
-      pieces: block.cid,
+      pieces,
     },
     proofs,
   })
-  invocation.attach(block)
```

Comment:
Needs an update to the spec since I believe it says the block MUST be attached to the invocation.

Reply:
it only mentions in aggregate/offer